### PR TITLE
Initial commit. Changelist -

### PR DESCRIPTION
**Pull request by abhiksingla**
1. Scaled-down initial states and actions
2. Support to set initial states from the configs
3. Bypassed data loading and processing with static datasets collected from offline pipelines
4. Differential state prediction
5. Added New Moab datasets

# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Reviewer Tasks

List here all the tasks the reviewer needs to review:

- [ ] Test `examples/`

**Test Configuration**:
* OS:
* Python Version:
* Bonsai SDK Version:

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by abhiksingla** [comment]
Hyperparameters worked best for me:

<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>

<meta http-equiv="Content-Style-Type" content="text/css">
<title></title>
<meta name="Generator" content="Cocoa HTML Writer">
<meta name="CocoaVersion" content="2113.4">
<style type="text/css">
p.p1 {margin: 0.0px 0.0px 0.0px 0.0px; font: 13.0px 'Helvetica Neue'}
table.t1 {border-collapse: collapse}
td.td1 {border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #9a9a9a #9a9a9a #9a9a9a #9a9a9a; padding: 1.0px 5.0px 1.0px 5.0px}
</style>
</head>
<body>


Name | DDM6 | DDM7 | DDM9 | DDM10 | DDM11
-- | -- | -- | -- | -- | --
Data | 500k mix | 19k expert | 5k expert | 19k medium | 5k medium
Model | Xgboost | Xgboost | Xgboost | Xgboost | Xgboost
Estimators | 100 | 100 | 100 | 100 | 100
Learning rate | 1 | 0.1 | 0.1 | 0.1 | 0.1
Depth | 5 | 4 | 4 | 5 | 4
Test ratio | 0.001 | 0.001 | 0.001 | 0.001 | 0.001


</body>
</html>
